src/main/config/config.c

```diff
@@ -778,8 +780,9 @@ void activateConfig(void)
     );
 
     gyroUseConfig(&masterConfig.gyroConfig, currentProfile->pidProfile.gyro_soft_lpf_hz);
+	gyroUseNotchConfig(currentProfile->pidProfile.gyro_soft_notch_hz, currentProfile->pidProfile.gyro_soft_notch_cutoff_hz)
```

Comment:
I considered it but I didn't want to change an existing function as it might break unit tests so I made a new one